commands/import.go

```diff
@@ -314,3 +329,112 @@ func jobListCmdF(c client.Client, command *cobra.Command, jobType string) error
 func importJobListCmdF(c client.Client, command *cobra.Command, args []string) error {
 	return jobListCmdF(c, command, model.JobTypeImportProcess)
 }
+
+type Statistics struct {
+	Schemes        int `json:"schemes"`
+	Teams          int `json:"teams"`
+	Channels       int `json:"channels"`
+	Users          int `json:"users"`
+	Emojis         int `json:"emojis"`
+	Posts          int `json:"posts"`
+	DirectChannels int `json:"direct_channels"`
+	DirectPosts    int `json:"direct_posts"`
+	Attachments    int `json:"attachments"`
+}
+
+func importValidateCmdF(command *cobra.Command, args []string) error {
+	configurePrinter()
+
+	defer printer.PrintT("Validation complete\n", struct {
+		Completed bool `json:"completed"`
+	}{true})
```

Comment:
Is this template required?

Reply:
I'd like to output a confirmation that the validation is done. I could turn it into just a print,  you are right.